gaas-rest/pom.xml

```diff
@@ -55,6 +56,11 @@
             <artifactId>graph</artifactId>
             <version>${gaffer.version}</version>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.gchq.gaffer</groupId>
+            <artifactId>federated-store</artifactId>
+            <version>1.13.3</version>
```

Comment:
Version should be in properties. You seem to be using version `1.13.3` and `1.13.0` of `Gaffer`. It gets inherited by the parent pom so you may not even need versions. We have just moved to `1.16.0` so may want to use that.

Reply:
Done